hexrd/grainmap/nfutil.py

```diff
@@ -1262,7 +1262,13 @@ def gen_trial_exp_data(grain_out_file,det_file,mat_file, mat_name, max_tth, comp
     # crystallography data
     beam_energy = valunits.valWUnit("beam_energy", "energy", instr.beam_energy, "keV")
     beam_wavelength = constants.keVToAngstrom(beam_energy.getVal('keV'))
-    dmin = valunits.valWUnit("dmin", "length",
+    if max_tth is not None:
+        print('ping')
```

Comment:
Maybe you should remove this debug statement? Or you can leave it if you prefer. It's up to you.